core/src/main/scala/org/apache/spark/internal/config/ConfigProvider.scala

```diff
@@ -66,7 +66,7 @@ private[spark] class SparkConfigProvider(conf: JMap[String, String]) extends Con
     findEntry(key) match {
       case e: ConfigEntryWithDefault[_] => Option(e.defaultValueString)
       case e: ConfigEntryWithDefaultString[_] => Option(e.defaultValueString)
-      case e: FallbackConfigEntry[_] => defaultValueString(e.fallback.key)
+      case e: FallbackConfigEntry[_] => get(e.fallback.key)
```

Comment:
What issue? The code you're commenting on does not exist in 1.6. If you're having issues, please ask questions on the mailing lists or use the bug tracker.